contracts/ConvictionVoting.sol

```diff
@@ -279,6 +292,23 @@ contract ConvictionVoting is AragonApp, TokenManagerHook {
         return voterStakedProposals[_voter];
     }
 
+    /**
+    * @dev IDisputable interface conformance
+    */
+    function canChallenge(uint256 _proposalId) external view returns (bool) {
+        return proposals[_proposalId].proposalStatus == ProposalStatus.Active;
```

Comment:
Even though we are not creating an agreement action for `ABSTAIN_PROPOSAL` , should we add the  `_proposalId != ABSTAIN_PROPOSAL_ID` check ?

Reply:
I don't think that's necessary.